typescript/packages/plugins/allora/src/allora.plugin.ts

```diff
@@ -0,0 +1,22 @@
+import { PluginBase } from '@goat-sdk/core'
+import { AlloraService } from './allora.service'
+import { PricePredictionTimeframes } from './types'
+
+interface AlloraPluginOptions {
+    apiKey: string
+    network: 'mainnet' | 'testnet'
```

Comment:
maybe `localnet` too

Reply:
Is this needed? Seems like it's not being used :)

Reply:
not needed at all, but could be good for those doing intense local testing, running Allora chain locally

Reply:
Replaced with `rpcEndpoint`